src/plugin.js

```diff
@@ -51,28 +51,37 @@ export const handleEncryptedEvent = (event, options, sessions, player) => {
     return;
   }
 
-  // "Initialization Data must be a fixed value for a given set of stream(s) or media
-  // data. It must only contain information related to the keys required to play a given
-  // set of stream(s) or media data."
-  // eslint-disable-next-line max-len
-  // @see [Initialization Data Spec]{@link https://www.w3.org/TR/encrypted-media/#initialization-data}
-  if (hasSession(sessions, event.initData)) {
-    // TODO convert to videojs.log.debug and add back in
-    // https://github.com/videojs/video.js/pull/4780
-    // videojs.log('eme',
-    //             'Already have a configured session for init data, ignoring event.');
-    return;
-  }
+  let initData = event.initData;
 
-  sessions.push({ initData: event.initData });
+  return getSupportedKeySystem(options.keySystems).then(({keySystem}) => {
+    if (options.keySystems[keySystem] &&
+        options.keySystems[keySystem].pssh) {
+      initData = options.keySystems[keySystem].pssh;
```

Comment:
Might be good to leave a comment here mentioning why it's useful to use the provided pssh (that it can save requests)